### PR TITLE
Add Konnect signup CTA

### PR DESCRIPTION
**Pull request by fabianrbz**
### Description

[Jira ticket](https://konghq.atlassian.net/browse/DOCU-3003)

Add Konnect Signup CTA to Docs and remove A/B test for "Personalized Demo".

### Testing instructions

Netlify link: 

### Checklist 

- [ ] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_includes/nav-v2.html [hunk 1]
```diff
@@ -90,7 +90,10 @@
         </li>
       </ul>
       <a id="top-cta" href="https://incubator.konghq.com/?utm_source=docs.konghq.com" class="navbar-button" target="_blank">
-        Early Access
+        Personalized Demo
```
Comment (acgoldsmith) [inline app/_includes/nav-v2.html:93]:
```suggestion
        Get A Demo
```

Comment (acgoldsmith) [reply app/_includes/nav-v2.html:93]: @fabianrbz had a parallel request to change text on this button, so adding in here as proposed change.

**Review by acgoldsmith — APPROVED** [review]
LGTM for Konnect CTA; added a comment to update demo text, if you'll just check to ensure that was done correctly, then good to merge. Thanks, @fabianrbz

**Comment by fabianrbz** [comment]
@acgoldsmith I added the change and it looks good, this is ready to be merged!

**Pull request merged**